docs/network/evm-nodes.md

```diff
@@ -38,7 +59,7 @@ The following instructions use the variable `<EVM_DATA_DIR>` to represent this d
 
    ```bash
    octez-evm-node init config \
-     --network mainnet \
+     --network testnet \
```

Comment:
Any reason why you want to replace mainnet with testnet here?

Reply:
Before, some examples were testnet and some were mainnet. I'm making them all testnet for consistency.

Reply:
:+1: